### PR TITLE
Refactor SendUsersAdminController

### PR DESCRIPTION
**Pull request by AbdullahAli**
Add some tests and refactor code 😚

### CODE REVIEW

File: app/models/userModel.js [hunk 1]
```diff
@@ -138,6 +138,10 @@ const userSchema = new Schema({
 			required: 'all permissions must be set'
 		}
 	}
+}, {
+	toObject: {
+		transform: (_, record) => { delete record.accessKey; }
```
Comment (AbdullahAli) [inline app/models/userModel.js]: Remove `accessKey` on the model directly so we don't have to `omit` everywhere 

giphy 2

Comment (keirog) [reply app/models/userModel.js]: 😻

File: test/controllers/send-users-admin.controller.test.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+const { mockReq, mockRes } = require('sinon-express-mock');
+const SendUsersAdminController = require('../../app/controllers/send-users-admin.controller');
+const validUserFixture = require('../fixtures/validUserFixture.json');
+const { tearDownDb } = require('../../fakery/users.fakery');
+const { omit } = require('lodash');
+const chai = require('chai');
+const expect = chai.expect;
+chai.use(require('chai-as-promised'));
+
+describe.only('Send Users Admin Controller', () => {
```
Comment (keirog) [inline test/controllers/send-users-admin.controller.test.js]: 😱

File: test/controllers/send-users-admin.controller.test.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+					const getUserInfoStub = sinon.stub(User, 'getUserInfo').resolves(expectedResult);
+
+					await makeRequest();
+
+					expect(getUserInfoStub).calledWith(validUserFixture.accessKey);
+					expect(this.res.json.calledWith(expectedResult)).to.be.true;
+				});
+			});
+
+			describe('and user record does not exists', () => {
```
Comment (keirog) [inline test/controllers/send-users-admin.controller.test.js]: Minor: ~exists~ exist

Comment (AbdullahAli) [reply test/controllers/send-users-admin.controller.test.js]: Good spot 👍 fixed: https://github.com/Financial-Times/email-send-api/pull/279/commits/05bb3a2c4afca6c8826285b621f05acfa38acae1

**Comment by AbdullahAli** [comment]
>When that .only’s gone

There isn't any `only`

tenor

File: test/controllers/send-users-admin.controller.test.js (hunk 2 above)
Comment (AbdullahAli) [reply test/controllers/send-users-admin.controller.test.js]: fixed: https://github.com/Financial-Times/email-send-api/pull/279/commits/a4647a09b553514ba40a93a27d0cd25c1ae897f5 😆

**Comment by AbdullahAli** [comment]
Closing this for now since it was been stale to avoid any unnecessary bugs 🐞

**Pull request closed without merging**